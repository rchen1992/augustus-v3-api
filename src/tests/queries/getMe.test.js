const { testQuery } = require('@tests/utils');

const currentUser = {
    user_id: '10',
    user_name: 'newboi',
    email: 'newboi@gmail.com',
    created_at: '2018-12-11 09:39:31',
    updated_at: '2018-10-09 01:00:11',
};

testQuery({
    id: 'Get current user',
    query: `
        {
            me {
                id
                userName
                email
                createdAt
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            currentUser,
        };
    },
    expected: undefined,
});
