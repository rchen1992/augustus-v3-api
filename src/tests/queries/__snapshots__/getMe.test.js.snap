// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get current user 1`] = `
Object {
  "data": Object {
    "me": Object {
      "createdAt": "2018-12-11 09:39:31",
      "email": "newboi@gmail.com",
      "id": "10",
      "userName": "newboi",
    },
  },
}
`;
